src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -293,6 +300,25 @@ export const LocatorEditDialog: React.FC<Props> = ({
           ]}
         />
       </FormItem>
+      <FormItem
+        wrapperCol={{ span: 24, xs: { offset: 0 }, sm: { offset: 4 } }}
+        name="annotationType"
+        style={{ marginBottom: "8px" }}
+      >
+        <Select
+          disabled={!isValidLocator(validationMessage)}
```

Comment:
I couldn't find a requirement about disabling this select in ticket. Should we do it?

Reply:
Thanks for pointing that out, removed